comments.php

```diff
@@ -1,82 +0,0 @@
-<?php
-/**
- * The template for displaying comments.
- *
- * The area of the page that contains both current comments
- * and the comment form. The actual display of comments is
- * handled by a callback to go_comment() which is
- * located in the functions.php file.
- *
- * @package Go
- */
-
-/*
- * If the current post is protected by a password and
- * the visitor has not yet entered the password we will
- * return early without loading the comments.
- */
-if ( post_password_required() ) {
-	return;
-}
-?>
-
-<div class="comments-area__wrapper">
-	<div id="comments" class="comments-area max-w-base m-auto">
-
```

Comment:
Can you clarify a few things for me? You may already know what happens in some instances with the new SE themes. 
- What happens with comments now? I know there are comments blocks and things like that, but that is all built-in now?
- Also I notice we are removing the archive page. Is that something that is more built-in now also?
- We removed the 404 page. Is that the same deal with SE where it's kind of built-in?

Reply:
- Comment listing and form are handled by core blocks now.
- We will create a new archive template with blocks.
- We will create a new 404 page with blocks.

Missing page templates are something we will resolve before release along with individual block styling for core and CoBlocks blocks.